detox/.npmignore

```diff
@@ -15,9 +15,9 @@ wallaby.js
 __tests__
 __mocks__
 __snapshots__
-src/**/*.test.js
-src/**/*.mock.js
-src/**/*.mock.json
+**/*.test.js
```

Comment:
This is boy scouting.

When I checked whether `__tests__/helpers.js` gets into the final npm package, I found that we were not excluding some test files from the published package, e.g.: `local-cli/build.test.js`, etc.